static/app/views/monitors/components/overviewTimeline/index.tsx

```diff
@@ -47,7 +52,8 @@ export function OverviewTimeline({monitorList}: Props) {
           <SegmentedControl.Item key="30d">{t('Month')}</SegmentedControl.Item>
         </SegmentedControl>
       </ListFilters>
-      <TimelineScrubber />
+      <GridLineTimeLabels resolution={resolution} end={nowRef.current} />
+      <GridLineOverlay resolution={resolution} end={nowRef.current} />
```

Comment:
for the overlay I just used an absolute positioned div in grid-column: 2 and starting at grid-row: 1 with height/width: 100% so it spans the whole right side of the grid. Let me know if theres a better way to do this.